src/main/resources/templates/sprint/sprint-add.html

```diff
@@ -35,8 +35,8 @@ <h1>Add sprint</h1>
     </div>
 
     <div class="col-12">
-        <button type="submit" class="btn btn-secondary" id="cancel" name="cancel">Cancel</button>
         <button type="submit" class="btn btn-primary" id="save" name="save">Save</button>
+        <button type="submit" class="btn btn-secondary" id="cancel" name="cancel">Cancel</button>
```

Comment:
revert this change. cancel button should be first. and the action button should be second.
also, avoid modifying files outside of your task scope. stick to edit sprint task.